Automattic/woocommerce-payments
# Remove WooCommerce as a composer dependency

~I set to update the Jetpack Autoloader to v2, and because nothing is ever easy, I found some problems along the way.~

I ended up removing our `woocommerce/woocommerce` dependency from `composer.json`. It was only used for testing, and it was causing more problems than it was worth. Instead, I rewrote the `install-wp-tests.sh` script to install WooCommerce from wp.org instead of copying from our `vendor` folder. I think it's even a bit more readable now.

#### Testing instructions

- Delete your `vendor` folder.
- Run `composer install` and `npm start`.
- See that the plugin works correctly (a quick smoke test like going to the Deposits page should be enough).
- See that the tests on Travis CI pass.
- Run `bin/install-wp-tests.sh`, note that WooCommerce is installed in the testing WP site.
- Run `composer run test`, make sure tests pass.
